packages/nux/src/components/guide/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Guide renders when there are pages 1`] = `
```

Comment:
I don't like snapshot testing personally, I feel we just end up updating snapshots when the component change.

Reply:
We could instead look for the Pages component and just check that their length is one.

Reply:
I like that having to update the snapshot when a component changes means that the reviewer can see exactly how the markup has changed in the PR's diff.

But you're the boss! 😀 removed in 809d0861f.